### DISCUSSION TITLE
Prácticas Intro

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#23] [Q&A] [answered]
Hola, buenos días tengo una duda de las prácticas. ¿Nosotros debemos visualizar como sería?
Me refiero a que debemos poner datos según nosotros creamos. Obviamente que tenga sentido y vaya conforme al caso. Oh debemos investigar y hacer más como encuestas o algo por el estilo.

**Comment by AngelCruzO** [comment, marked as answer]
Respondiendo a tu duda si, lo que tenemos que hacer es mostrar desde nuestro punto de vista como es que se debe ver el sistema completo. Lo primero que se debe realizar es levantar los requisitos, que en otras palabras es encontrar en el enunciado los puntos clave que cumplan las necesidades del cliente.

Los datos en un principio pueden ser al criterio de cada quien, conforme se revisan los requisitos se van puliendo todos esos detalles; tambien en algunas ocasiones el cliente es el que decide que datos lleva su sistema.

En el caso de realizar formularios no lo veo tan necesario por el tiempo de entrega, lo que te recomiendo es  hacer una breve investigación sobre como se estructura un caso juridico.

**Reply by GloriaCoralCerecedo** [reply]
Oh vale si eso me suponía, es en base a nuestro punto de vista, muchas gracias.

**Reply by AngelCruzO** [reply]
de nada

**Reply by Ingrzs** [reply]
En la práctica 2 el buyer trae predefinido unos datos así que yo me estoy basando en esos datos de acuerdo a la descripción. Saludos.

**Reply by AngelCruzO** [reply]
@Ingrzs Pues para los datos de usuario si, pero aun te faltan los datos que se manejan en un proceso juridico. Se tienen que considerar todos los datos que maneja un sistema

**Reply by Ingrzs** [reply]
Exacto , en lo personal ya eso lo abarcaria en público objetivo, porque el buyer trae como dato, menor de edad y eso limita algunas cosas de lo que puede hacer...

**Reply by AngelCruzO** [reply]
Asi es, porque el bayer persona es especifico. Ya el publico objetivo es general y es una muestra para otro tipo de estudio
